Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/WindowsXamlHostBaseExt.cs

```diff
@@ -23,7 +23,7 @@ protected WindowsXamlHostBaseExt(string typeName)
         internal virtual void InitializeElement()
```

Comment:
Don't call this method from `.ctor`

When the virtual method is called, the actual type that executes the method is not selected until later at runtime. So when a ctor calls a virtual, it is possible that the type has not yet been selected, or its ctor has not yet executed. The ctor should be redesigned to not call the virtual, or the method should not be made virtual.